include/swift/AST/Attr.def

```diff
@@ -718,7 +718,7 @@ DECL_ATTR(exclusivity, Exclusivity,
   128)
 
 DECL_ATTR(_backDeploy, BackDeploy,
-  OnAbstractFunction |
+  OnAbstractFunction | OnAccessor | OnSubscript | OnVar |
```

Comment:
Nitpick: can we add a static assert to ensure `_backDeploy` and `_alwaysEmitIntoClient` can operate on the same kind of declarations?

Reply:
OK, sounds reasonable to me!